lib/crawler_web/controllers/keyword_controller.ex

```diff
@@ -0,0 +1,72 @@
+defmodule CrawlerWeb.KeywordController do
+  use CrawlerWeb, :controller
+
+  alias Crawler.Keyword.Helpers.CSVParser
+  alias Crawler.Keyword.Keywords
+  alias Crawler.Keyword.Schemas.KeywordCSVFile
+
+  def create(conn, %{"keyword_csv_file" => params}) do
+    changeset = %{
+      KeywordCSVFile.create_changeset(%KeywordCSVFile{}, params)
+      | action: :validate
+    }
+
+    case changeset.valid? do
+      true ->
+        parse_uploaded_file(conn, changeset.changes)
```

Comment:
we do both action in that method `parse and creating` shall we name it as `process_....` method

```suggestion
        process_uploaded_file(conn, changeset.changes)
```

Reply:
Updated in [1d816b2](https://github.com/vnntsu/elixir-ic/pull/36/commits/1d816b234f96b49e84b553a585e8119e4990ed31)